src/ObserverPattern/Observer/IObserver.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ObserverPattern.Observer
+{
+    public interface IObserver<in T>
+    {
+        void Update(T account, string message);
```

Comment:
parameter probably shouldn't be named "account" as the interface can be used in a lot of different scenarios

Reply:
I was wondering if it would be better to have event objects and this Update method would therefore receive events.

Reply:
Yeah the name is wrong. It could receive events, nice idea. I'll do an implementation with that approach.